edi-module/Package.md

```diff
@@ -0,0 +1,5 @@
+Prints "Hello, World!" with a hello function.
+[//]: # (above is the package summary)
```

Comment:
Shall we remove the above two lines and the package.md needs to be more descriptive. Consider following an existing module in ballerina-platform and update